server/methods/accounts/accounts.js

```diff
@@ -59,6 +60,74 @@ export function verifyAccount(email, token) {
   return false;
 }
 
+
+/**
+ * updateEmailAddress
+ * @summary update a users email address
+ * @param {String} email - user email
+ * @returns {Boolean} - returns boolean
+ */
+export function updateEmailAddress(email) {
+  check(email, String);
+
+  // Get users current Email address
+  const user = Meteor.user();
+
+  // Add email to user account
+  MeteorAccounts.addEmail(user._id, email);
+
+  return true;
+}
+
+
+/**
+ * removeEmailAddress
+ * @summary revmoe a users email address
+ * @param {String} email - user email
+ * @returns {Boolean} - returns boolean
+ */
+export function removeEmailAddress(email) {
+  check(email, String);
+
+  // Get user
+  const user = Meteor.user();
+
+  // Remove email address from user
+  MeteorAccounts.removeEmail(user._id, email);
+
+  // Verify new address
+  sendUpdatedVerificationEmail(user._id);
+
+  // Sync users and accounts collections
+  syncUsersAndAccounts();
```

Comment:
I'm not sure this is the best way to sync `users` and `accounts`... any input would be appreciated